Modules/_lsprof.c

```diff
@@ -929,33 +932,39 @@ profiler_dealloc(ProfilerObject *op)
     Py_DECREF(tp);
 }
 
+/*[clinic input]
+_lsprof.Profiler.__init__
+
+    timer: object(c_default='NULL') = None
+    timeunit: double = 0.0
+    subcalls: bool = True
+    builtins: bool = True
+
+Builds a profiler object using the specified timer function.
+
+The default timer is a fast built-in one based on real time.
+For custom timer functions returning integers, timeunit can
+be a float specifying a scale (i.e. how long each integer unit
```

Comment:
We try to avoid latin abbreviations in the docs.

```suggestion
be a float specifying a scale (that is, how long each integer unit
```